Lecture-6/Task1/l6task1/Application.java

```diff
@@ -0,0 +1,139 @@
+package l6task1;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Application {
+
+    private static String[] coffeeType = {"bean", "ground", "instant"};
```

Comment:
All constant fields should be `static final`
And should be named in UPPER_CASE

Reply:
done